ui/packages/atlasmap-core/src/contracts/common.ts

```diff
@@ -83,12 +83,19 @@ export enum FieldType {
 }
 
 export enum DocumentType {
+  CORE = 'Core',
+  CSV = 'CSV',
   JAVA = 'JAVA',
+  JSON = 'JSON',
+  JSON_INSTANCE = 'JSON_INSTANCE',
```

Comment:
Are the `SCHENA` and `INSTANCE` are required for something? We have `InspectionType` enum apart from this `DocumentType` at runtime.